app/views/prisoners/show.html.erb

```diff
@@ -71,4 +71,4 @@
 <%= render partial: 'shared/offence_info', locals: { editable_prd: true}  %>
 <%= render 'prison_allocation' %>
 <%= render 'community_information' %>
-<%= render 'shared/vlo_information' %>
+<%= render 'shared/vlo_information' if @prisoner.probation_record.present? %>
```

Comment:
Why can't VLO information be shown if the Probation Record is missing?

I thought VLOs were standalone.

Reply:
They are attached (for now) - this will hopefully go once the VLOs get moved

Reply:
### Todo

Once the VLO PR is merged in, this case change...